Siddhpant patch 1

siddhpant opened the pull request.

tmortagne commented on xwiki-rendering-syntaxes/xwiki-rendering-syntax-wikimodel/src/main/java/org/xwiki/rendering/internal/parser/wikimodel/WikiModelParserUtils.java:
```diff
@@ -58,21 +58,13 @@ public void parseInline(StreamParser parser, String content, Listener listener,
-                    } else {
-                        this.foundWord = true;
-                    }
+                    this.foundWord ? super.onWord(word) : this.foundWord = true;
```
I really don't like ternary operators which do two completely different things.

siddhpant replied:
It works exactly the same way, and shows the whole if-else block in one line. Effectively reduces the number of lines.

tmortagne commented on xwiki-rendering-syntaxes/xwiki-rendering-syntax-wikimodel/src/main/java/org/xwiki/rendering/internal/parser/wikimodel/WikiModelParserUtils.java:
```diff
@@ -58,21 +58,13 @@ public void parseInline(StreamParser parser, String content, Listener listener,
-                    } else {
-                        this.foundSpace = true;
-                    }
+                    this.foundSpace ? super.onSpace() : this.foundSpace = true;
```
Same comment.

siddhpant committed:
Update DefaultMacroManager.java

Fixed http://sonar.xwiki.org/issues/search#issues=a6a2701c-f668-4392-9fa6-13dfdf251466

siddhpant committed:
Update DoxiaPrinterAdapter.java

Fixed http://sonar.xwiki.org/issues/search#issues=6efc3f5c-99f8-4abb-bb3c-fabefba023c2

siddhpant committed:
Update MacroId.java

Fixed http://sonar.xwiki.org/issues/search#issues=b79f65b8-dea8-47d6-ada6-7921845ae614

siddhpant committed:
Update MacroId.java

siddhpant committed:
Update MacroId.java

siddhpant committed:
Update MacroId.java
